FSharpLu.Tests/JsonTests.fs

```diff
@@ -252,6 +268,20 @@ type JsonSerializerTests() =
         areReciprocal <| (Option.Some <| SomeAmbiguity.DUWithCaseWithFieldNamedSome.Some)
 
     [<TestMethod>]
+    [<TestCategory("FSharpLu.Json.CamelCase")>]
+    member __.``CamelCaseSerializer handles discriminated unions`` () =
+        let du = ComplexDu <| SomeField (2, 3)
+        let str = CamelCaseSerializer.serialize du
+        Assert.AreEqual("""{"complexDu":{"someField":[2,3]}}""", str)
+
+    [<TestMethod>]
```

Comment:
I'm currently struggling to create a way to make those tests without duplicating the whole `Reciprocality` static class... Basically parametrize it by serialize/deserialize function we want to use. I'm not sure if it is even possible with all these SRTP :)